package.json

```diff
@@ -16,6 +16,9 @@
     "typescript": "^5.2.2"
   },
   "devDependencies": {
-    "@rollup/plugin-yaml": "^4.1.2"
+    "@rollup/plugin-yaml": "^4.1.2",
+    "@types/datatables.net": "^1.10.28",
```

Comment:
Different version from the one in accession_id.astro (1.11.5) and the one in BaseLayout (2.0.8)

Works now but might cause issues later either because we look at the wrong versions of docs when doing something, or can't find the reason for weird behaviour because the library actually used is different from the one the editor highlights

The way I would normally expect this to work is that datatables gets declared as a dependency (only) here and then gets bundled into the website js by astro. This way if we want to check the version of it we're using we just need to look in the standard place + minor benefit of no extra connection (and no runtime dependency) to the astro cdn domain but that's mega minor compared to the banner swap content shift

Reply:
Need to check. Happy to hold off merging until we resolve the version and bundling issues. It is a prototype code and I'd like to fix the if missing viewable annotation_images etc so might be good to do that as well before merge.

Reply:
I've taken out the build and node dependencies in package.json, have also taken out script src from datasets and put it only in the BaseLayout instead. The npm build and import is problematic for some packages due to [reasons](https://docs.astro.build/en/guides/troubleshooting/) so not using that but just having one place (BaseLayout) seems to be the optimal solution for now.